docs/pages/get-started.mdx

```diff
@@ -25,7 +25,6 @@ We will run the following Teleport services:
 
 ```mermaid
 flowchart LR
-
 subgraph private[Private network]
```

Comment:
@ptgott I'm slightly concerned that this section of the graph describes a larger configuration than the page instructs.

Reply:
I thought it would make sense to show the Linux server within the context of a full Teleport deployment, since ultimately the intended user of the guide would be expanding their setup from the initial VM to protect more infrastructure resources. 

I also thought it would look a bit odd to only show the Linux server box.

What do you think?

Reply:
I think that the easy solve here is to add a note under the diagram explaining that it shows a scenario that extends beyond the contents of this page, and that adding additional resources is covered by the "Next steps" section.

Reply:
I'm going to remove the diagram from the Linux Server guide so we can tackle that in a separate PR!